config-model/src/main/java/io/strimzi/kafka/config/model/ConfigModel.java

```diff
@@ -28,6 +28,7 @@
     @JsonProperty("enum")
     private List<String> values;
     private String pattern;
+    private Object defaultVal;
```

Comment:
If this is really necessary, then let's at least call the property `defaultValue`.